server.js

```diff
@@ -181,33 +181,33 @@ async function getAccountsAndTransactions(userId, res) {
   // GET Accounts
   const accounts = await getAccounts(consumerApiPath, userId, accessToken);
 
-  accounts.forEach(account => {
+  for (const account of accounts) {
     const accountId = account.id;
     const accountBalance = account.balance;
-
+    
```

Comment:
not sure how picky you'd like to be with whitespacing, but there's some unneeded whitespacing here

Reply:
I think it's perfectly appropriate to be picky with whitespacing! 😁 

I'm updated the whitespacing using VS Code (which got some other lines too) in commit https://github.com/Banno/consumer-api-openid-connect-example/pull/5/commits/5642f4c1b4ae396150069e31a34e490b283b711d.